### PR TITLE
Use enso paths in cloud browser

### LINKED ISSUE #10947: Cloud explorer is putting the S3 link in not enso:
When a file is selected in the cloud browser it needs to insert the enso: link rather than the raw S3 link

### PR DESCRIPTION
**Pull request by farmaazon**
### Pull Request Description

Fixes #10947 

### Checklist

Please ensure that the following checklist has been satisfied before submitting the PR:

- [x] The documentation has been updated, if necessary.
- [x] Screenshots/screencasts have been attached, if there are any visual changes. For interactive or animated visual changes, a screencast is preferred.
- [x] All code follows the
      [Scala](https://github.com/enso-org/enso/blob/develop/docs/style-guide/scala.md),
      [Java](https://github.com/enso-org/enso/blob/develop/docs/style-guide/java.md),
      [TypeScript](https://github.com/enso-org/enso/blob/develop/docs/style-guide/typescript.md),
      and
      [Rust](https://github.com/enso-org/enso/blob/develop/docs/style-guide/rust.md)
      style guides. In case you are using a language not listed above, follow the [Rust](https://github.com/enso-org/enso/blob/develop/docs/style-guide/rust.md) style guide.
- [x] Unit tests have been written where possible.

### CODE REVIEW

File: app/gui2/src/components/widgets/FileBrowserWidget.vue [hunk 1]
```diff
@@ -114,17 +107,17 @@ function chooseFile(file: FileAsset) {
...
   isError.value ? error
-  : selectedFileDetails.isError.value ? selectedFileDetails.error
+  : currentUser.isError.value ? currentUser.error
   : undefined,
 )
 
-watch(selectedFileDetails.data, (details) => {
-  if (details) emit('pathSelected', details.file.path)
+watch(selectedFile, (file) => {
+  if (file && currentPath.value) emit('pathSelected', `${currentPath.value}/${file.title}`)
```
Comment (kazcw) [inline app/gui2/src/components/widgets/FileBrowserWidget.vue]: This will not work if a file is selected before `currentPath` becomes ready

**Review by kazcw — CHANGES_REQUESTED** [review]

File: app/gui2/src/components/widgets/FileBrowserWidget.vue (hunk 1 above)
Comment (farmaazon) [reply app/gui2/src/components/widgets/FileBrowserWidget.vue]: If currentPath is not ready, which is only case when currentUser is not ready, we should be in "busy" state.

But fixed this anyway, as it is now more error-proof.

Comment (kazcw) [reply app/gui2/src/components/widgets/FileBrowserWidget.vue]: Pending currentUser only causes busy state once a selection has been made.

**Review by kazcw — APPROVED** [review]

File: app/gui2/src/components/widgets/FileBrowserWidget.vue (hunk 1 above)
Comment (farmaazon) [reply app/gui2/src/components/widgets/FileBrowserWidget.vue]: Oh, you are right! The better I wrote the fix.

**Pull request merged**